application/org.openjdk.jmc.jolokia/src/main/java/org/openjdk/jmc/jolokia/preferences/messages.properties

```diff
@@ -0,0 +1,11 @@
+JolokiaPreferencePage_Description=Discover Jolokia Agents\n\nhttps://jolokia.org/reference/html/protocol.html\#discovery\n\n
```

Comment:
It'd be nice if the content of the string was similar to the other JVM Browser descriptions, see: https://github.com/search?q=repo%3Aopenjdk%2Fjmc%20%22these%20are%20the%20preferences%20for%22&type=code

"These are the preferences for [..]", maybe something like "These are the preferences for Jolokia Agent discovery"